docs/en/engines/database-engines/materialized-mysql.md

```diff
@@ -78,12 +78,15 @@ When working with the `MaterializedMySQL` database engine, [ReplacingMergeTree](
 | DATE, NEWDATE           | [Date](../../sql-reference/data-types/date.md)               |
 | DATETIME, TIMESTAMP     | [DateTime](../../sql-reference/data-types/datetime.md)       |
 | DATETIME2, TIMESTAMP2   | [DateTime64](../../sql-reference/data-types/datetime64.md)   |
+| YEAR                    | [String](../../sql-reference/data-types/string.md)           |
+| TIME                    | [String](../../sql-reference/data-types/string.md)           |
```

Comment:
This is also questionable, because it may be inconvenient to parse it back from String. As far as I understand, `TIME` in MySQL is similar to `Interval` data type in ClickHouse. But `Interval` is a special data type and it cannot be stored in tables. I'd prefer to map `TIME` to `Int64` and store number of microseconds.

Reply:
ok, YEAR-> UInt16 is good. but, TIME-> Int64 Will it confuse users ？the value in clickhouse looks different with value in mysql. But in terms of being able to parse, TIME-> Int64 is more advantageous.

Reply:
i will fix it later

Reply:
Yes, it will be confusing, but I think it's ok if we describe this behavior in the documentation

Reply:
And it will be possible to replace `Int64` with `Time64(6)` or `Interval` after #30935 is implemented

Reply:
done